modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,35 +6,69 @@
 [id="persistent-storage-csi-migration-sc-vsphere_{context}"]
 = vSphere automatic migration
 
+[IMPORTANT]
+====
+Migration includes significant consequences:
+
+* Enabling migration cannot be undone.
+
+* Migration can take awhile to complete depending on how many nodes are on the cluster.
+====
+
 == New installations of {product-title}
 For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
 
-[id="upgrading-openshift_{context}"]
-== Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+== Upgrading from {product-title} 4.13 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.13 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} upgrade can occur and automatic migration is enabled by default.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} upgrade by performing an administrator acknowledgment:
+
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.13-kube-1.27-api-removals-in-4.14":"true"}}' --type=merge
+----
 
 [IMPORTANT]
 ====
-Carefully review the following consequences before opting in to migration:
+If you do *not* update to vSphere 7.0 Update 3L or 8.0 Update 2 and use an administrator acknowledgment to upgrade to {product-title} 4.14, known issues can occur. link:https://access.redhat.com/node/7011683[Before proceeding with the administrator acknowledgement, carefully read this knowledge base article].
+====
 
-* link:https://access.redhat.com/node/7011683[There are known issues that can occur that can cause the migration to fail].
+== Upgrading from {product-title} 4.12 to 4.13
+If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 to 4.13, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} upgrade can occur and automatic CSI migration for vSphere only occurs if you opt in.
 
-* Enabling migration cannot be undone.
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} upgrade by:
 
-* Migration can take awhile to complete depending on how many nodes are on the cluster.
+* Initiating the upgrade from a {product-title} 4.12 Z-stream.
+
+* Performing an administrator acknowledgment by running the following command:
++
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-1.26-api-removals-in-4.14":"true"}}' --type=merge
```

Comment:
wrong key
cc @gnufied

Reply:
Why are we documenting 4.12 upgrades here though? Was it for EUS to EUS scenario?

Reply:
Also correct key is - `ack-4.12-kube-126-vsphere-migration-in-4.14`

Reply:
Strictly speaking we don't have to include this content here. I think that we were doing this to ensure absolute coverage by including it in the 4.12 and 4.13 docs.